play/src/i18n/fr-FR/chat.ts

```diff
@@ -44,9 +44,10 @@ const chat: DeepPartial<Translation["chat"]> = {
     searchUser: "Rechercher un utilisateur, une map, etc ...",
     searchChat: "Rechercher un canal, un message, etc ...",
     people: "Personnes",
-    rooms: "Salons",
+    rooms: "Vos Discussions",
     invitations: "Invitations",
     suggestedRooms: "Salons recommandés",
+    joinableRooms: "Salons disponibles",
```

Comment:
If we talk about "discussion," why use "Salons" here?
I think we need to choose between "Salons" or "Discussion"?